.circleci/config.yml

```
@@ -7,11 +7,22 @@ install_hex_rebar: &install_hex_rebar
       mix local.hex --force
       mix local.rebar --force
 
+install_system_deps: &install_system_deps
+  run:
+    name: Install system dependencies
+    command: |
+      apk add build-base
```

```suggestion
      apk add build-base git
```

I guess `build-base` doesn't have git after all